doc/manuals/ru.po

```diff
@@ -726,7 +726,7 @@ msgstr "СООБЩЕНИЕ ОБ ОШИБКАХ"
 #: src/security/cert_generators/file/security_file_certgen.8.in:167
 #: src/squid.8.in:261
 msgid ""
-"Bug reports need to be made in English.  See http://wiki.squid-cache.org/"
+"Bug reports need to be made in English.  See https://wiki.squid-cache.org/"
 "SquidFaq/BugReporting for details of what you need to include with your bug "
 "report."
 msgstr ""
```

Comment:
Please update the multiline URL that starts on the next line (below this change request).

Reply:
Done in [9e227de](https://github.com/squid-cache/squid/pull/1448/commits/9e227deea244a6b3e0a873a8b8f1e741c2f6fdfe).

Reply:
Sigh, missed the `git add` . Done now in [3859ef5](https://github.com/squid-cache/squid/pull/1448/commits/3859ef5317bdc82234912ed76b898be07dc640ab)